diff --git a/pymomentum/CMakeLists.txt b/pymomentum/CMakeLists.txt
index 71a247e..78ecb7f 100644
--- a/pymomentum/CMakeLists.txt
+++ b/pymomentum/CMakeLists.txt
@@ -10,11 +10,11 @@
 find_package(ATen CONFIG REQUIRED)
 find_package(Torch CONFIG REQUIRED)
 
-find_library(torch_python
-  NAMES torch_python
-  HINTS $ENV{CONDA_PREFIX}/lib/python3.12/site-packages/torch/lib/
-  REQUIRED
-)
+# find_library(torch_python
+#   NAMES torch_python
+#   HINTS $ENV{CONDA_PREFIX}/lib/python3.12/site-packages/torch/lib/
+#   REQUIRED
+# )
 
 #===============================================================================
 # Find build dependencies
