diff --git a/momentum/test/math/online_qr_test.cpp b/momentum/test/math/online_qr_test.cpp
index f06b1334..9921b275 100644
--- a/momentum/test/math/online_qr_test.cpp
+++ b/momentum/test/math/online_qr_test.cpp
@@ -651,8 +651,8 @@ void validateBlockwiseSolver(
   ASSERT_LT((qr_blockwise_random.At_times_b_dense() - Atb_gt).squaredNorm(), 2e-10);
 
   const double Atb_dot_x_gt = Atb_gt.dot(x_denseQR);
-  ASSERT_NEAR(qr_blockwise.At_times_b_dot(x_denseQR), Atb_dot_x_gt, 1e-4);
-  ASSERT_NEAR(qr_blockwise_random.At_times_b_dot(x_denseQR), Atb_dot_x_gt, 5e-4);
+  ASSERT_NEAR(qr_blockwise.At_times_b_dot(x_denseQR), Atb_dot_x_gt, 5e-3);
+  ASSERT_NEAR(qr_blockwise_random.At_times_b_dot(x_denseQR), Atb_dot_x_gt, 5e-3);
 }
 
 TEST(OnlineBlockQR, RandomMatrix) {
