diff --git a/setup.py b/setup.py
index beba68cf..b337d338 100644
--- a/setup.py
+++ b/setup.py
@@ -49,7 +49,7 @@ class CMakeBuild(build_ext):
             "-DMOMENTUM_BUILD_PYMOMENTUM=ON",
             "-DMOMENTUM_BUILD_TESTING=ON",  # TODO: OFF when pym.geometry doesn't depend on testing
             "-DMOMENTUM_BUILD_EXAMPLES=OFF",
-            "-DMOMENTUM_ENABLE_SIMD=ON",
+            f"-DMOMENTUM_ENABLE_SIMD={os.environ.get('MOMENTUM_ENABLE_SIMD', 'OFF')}",
             "-DMOMENTUM_USE_SYSTEM_RERUN_CPP_SDK=ON",
         ]
         build_args = ["--target", os.path.basename(ext.name)]
