diff --git a/momentum/character_solver/gauss_newton_solver_qr.cpp b/momentum/character_solver/gauss_newton_solver_qr.cpp
index 32fe8a9..16980dc 100644
--- a/momentum/character_solver/gauss_newton_solver_qr.cpp
+++ b/momentum/character_solver/gauss_newton_solver_qr.cpp
@@ -10,7 +10,7 @@
 #include "momentum/character_solver/skeleton_error_function.h"
 #include "momentum/common/profile.h"
 
-#include <gsl/gsl_util>
+#include <gsl/util>
 
 namespace momentum {
 
