diff --git a/setup.py b/setup.py
index 7861bbd4..e192d8ea 100644
--- a/setup.py
+++ b/setup.py
@@ -57,10 +57,10 @@ class CMakeBuild(build_ext):
             cmake_args += [item for item in os.environ["CMAKE_ARGS"].split(" ") if item]
 
         # Default to Ninja
-        if "CMAKE_GENERATOR" not in os.environ:
-            cmake_args += ["-G Ninja"]  # TODO: Select by platform
-        elif self.compiler.compiler_type == "msvc":
+        if self.compiler.compiler_type == "msvc":
             cmake_args += ["-G Visual Studio 17 2022"]
+        elif "CMAKE_GENERATOR" not in os.environ:
+            cmake_args += ["-G Ninja"]
 
         if sys.platform.startswith("darwin"):
             # Cross-compile support for macOS - respect ARCHFLAGS if set
