From 043c70c7c9331af07a04f66b9027643bc425e188 Mon Sep 17 00:00:00 2001
From: Jeongseok Lee <jeongseok@meta.com>
Date: Wed, 19 Mar 2025 08:24:24 -0700
Subject: [PATCH] Disable SIMD by default for pymomentum

---
 pyproject.toml | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index ddbfb96d..f2e1e4c4 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,7 +24,8 @@ cmake.args = [
     "-DMOMENTUM_BUILD_PYMOMENTUM=ON",
     "-DMOMENTUM_BUILD_EXAMPLES=OFF",
     "-DMOMENTUM_BUILD_TESTING=ON",
-    "-DMOMENTUM_USE_SYSTEM_GOOGLETEST=ON",
+    "-DMOMENTUM_ENABLE_SIMD=OFF",
+    "-DMOMENTUM_USE_SYSTEM_GOOGLETEST=OFF",
     "-DMOMENTUM_USE_SYSTEM_PYBIND11=OFF",
     "-DMOMENTUM_USE_SYSTEM_RERUN_CPP_SDK=ON",
 ]
@@ -48,6 +49,7 @@ cmake.args = [
     "-DMOMENTUM_BUILD_PYMOMENTUM=ON",
     "-DMOMENTUM_BUILD_EXAMPLES=OFF",
     "-DMOMENTUM_BUILD_TESTING=ON",
+    "-DMOMENTUM_ENABLE_SIMD=OFF",
     "-DMOMENTUM_USE_SYSTEM_GOOGLETEST=ON",
     "-DMOMENTUM_USE_SYSTEM_PYBIND11=ON",
     "-DMOMENTUM_USE_SYSTEM_RERUN_CPP_SDK=ON",
-- 
2.20.1.windows.1

